ledger-core/insolar/payload/marker_auth.go

```diff
@@ -52,3 +52,9 @@ func (m *VDelegatedCallRequest) customSubject() (Reference, authSubjectMode) {
 func (m *VDelegatedCallResponse) customSubject() (Reference, authSubjectMode) {
 	return m.GetCallee(), hasAuthSubject
 }
+
+// It returns hasAuthSubject ( i.e. for current pulse ) since VDelegatedRequestFinished can come only with
```

Comment:
To be "go doc" it should start with method name
// customSubject returns...

Reply:
Fixed